core/gui/nx.json

```diff
@@ -28,5 +17,22 @@
     "@schematics/angular:directive": {
       "prefix": "texera"
     }
-  }
+  },
+  "namedInputs": {
+    "default": ["{projectRoot}/**/*", "sharedGlobals"],
+    "sharedGlobals": [],
+    "production": ["default"]
+  },
+  "targetDefaults": {
+    "build": {
+      "inputs": ["production", "^production"],
+      "cache": true
+    },
+    "lint": {
+      "cache": true
+    }
+  },
+  "nxCloudAccessToken": "Njk2ODE5NjMtMmRjNy00NjNmLWEwMDAtZGVkZTMwMmMwM2M2fHJlYWQtd3JpdGU=",
```

Comment:
good catch. after some reading, I believe this is a private key. Since this key has been in version control for a long while (long before this PR), I will not change it in this PR. Instead, I will recycle this access token and regenerate one, then find a good measure to use it without committing the token to version control.